pkg/create/templates/docker.go

```diff
@@ -3,38 +3,56 @@ package templates
 // Dockerfile is a default Dockerfile laid down by s2i create
 const Dockerfile = `
 # {{.ImageName}}
-FROM openshift/base-centos7
```

Comment:
I'm not sure it makes more sense for bootstrap to output a working nginx example compared to its previous behaviour of outputting all the required boilerplate and TODOs marked.  If people want a working nginx example, that's in the examples directory.  Others - thoughts?